test/convex-platform/BoosterLite.spec.ts

```diff
@@ -0,0 +1,289 @@
+import { expect } from "chai";
+import { BigNumber, Signer } from "ethers";
+import hre, { ethers } from "hardhat";
+import { Account } from "types";
+import { CanonicalPhaseDeployed, SidechainDeployed } from "../../scripts/deploySidechain";
+import { DeployL2MocksResult } from "../../scripts/deploySidechainMocks";
+import { increaseTime, increaseTimeTo, simpleToExactAmount } from "../../test-utils";
+import { DEAD_ADDRESS, ZERO, ZERO_ADDRESS } from "../../test-utils/constants";
+import { impersonateAccount } from "../../test-utils/fork";
+import { sidechainTestSetup } from "../../test/sidechain/sidechainTestSetup";
+import { BaseRewardPool__factory, BoosterLite, ERC20__factory } from "../../types/generated";
+
+type Pool = {
+    lptoken: string;
+    token: string;
+    gauge: string;
+    crvRewards: string;
+    stash: string;
+    shutdown: boolean;
+};
+
+const NATIVE_FEE = simpleToExactAmount("0.2");
+const L1_CHAIN_ID = 111;
+const L2_CHAIN_ID = 222;
+
+// ADD TESTS TO VERIFY WRONG STORAGE DUE TO BAD srcs
+//  @method Booster.earmarkRewards: Error: VM Exception while processing transaction: reverted with reason string 'LayerZeroMock: not enough native for fees'
+// @method BaseRewardPool.processIdleRewards():
+// Error: VM Exception while processing transaction: reverted with reason string 'LayerZeroMock: not enough native for fees'
+// @method BaseRewardPool.stake:
+// Error: VM Exception while processing transaction: reverted with reason string 'RewardPool : Cannot stake 0'
+describe("BoosterLite", () => {
```

Comment:
Should we add some tests for initialise, "Initialise cannot be called twice"

Reply:
Done